ports/spellsling/README.md

```diff
@@ -0,0 +1,14 @@
+## Notes
+Thanks to [youdoyoubuddy] (https://youdoyoubuddy.itch.io/) for creating this game and making it available for free! 
```

Comment:
This link wont work with a space between the close square bracket and open parenthesis. `] (`. :)

Reply:
fixed